.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Thanks for your contribution. It must be reviewed by a member of the [`cockroachdb/docs-ext-pr-reviewers`](https://github.com/orgs/cockroachdb/teams/docs-ext-pr-reviewers) team. Before asking for review, add a description of your change and complete the following (add an `x` between the brackets):
+
+- [ ] Successful local build (`make standard`)
+- [ ] Successful local linkcheck (`make linkcheck`)
```

Comment:
I think having a local environment and running these should be recommended but not required. It's probably too much overhead for someone who wants to make minor or moderate doc improvements. Also, we're running these checks on the PR for external contributors too, right? So can we just say that if the tests don't pass, you'll need to run them locally to view details, and if they're not willing/able to do that, it may take a while to get assistance and get the content merged. WDYT @mdlinville , @nickvigilante ?

Reply:
i don't know, I think for a non-docs contribution, the PR author would be expected to run and pass the tests locally before creating the PR. If the contributor isn't willing to test their change, it seems like they might not be holding up their end of the deal. WDYT?

Reply:
After thinking about this more, our CONTRIBUTING guide is tailored specifically for macOS, so I think until we can account for Windows machines within that guide, it's fair to assume they may not know how to install all dependencies required in order to get a link check to pass. We may be required to check those things for them and offer suggestions prior to merging.